ContiguousArray.java

```diff
@@ -0,0 +1,44 @@
+/* Leetcode Problem - 525
+https://leetcode.com/problems/contiguous-array/
+ Time Complexity : O(n)
+ Space Complexity : O(1)
```

Comment:
O(n), The length of the input array is not fixed hence the running sum and space of hashmap is not constant.